transaction/core/tests/validation.rs

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
 // Copyright (c) 2022 The MobileCoin Foundation
```

Comment:
Nit: Duplicate copyright

```suggestion
// Copyright (c) 2018-2022 The MobileCoin Foundation
```

Reply:
whoops, thanks, i think my hook did this